DUI3-DX/Converters/ArcGIS/Speckle.Converters.ArcGIS3/Layers/VectorLayerToHostConverter.cs

```diff
@@ -0,0 +1,110 @@
+using ArcGIS.Core.Data;
+using ArcGIS.Core.Data.DDL;
+using ArcGIS.Core.Data.Exceptions;
+using ArcGIS.Core.Geometry;
+using ArcGIS.Desktop.Framework.Threading.Tasks;
+using ArcGIS.Desktop.Mapping;
+using Objects.GIS;
+using Speckle.Converters.ArcGIS3.Utils;
+using Speckle.Converters.Common;
+using Speckle.Converters.Common.Objects;
+using Speckle.Core.Models;
+using FieldDescription = ArcGIS.Core.Data.DDL.FieldDescription;
+
+namespace Speckle.Converters.ArcGIS3.Layers;
+
+[NameAndRankValue(nameof(VectorLayer), NameAndRankValueAttribute.SPECKLE_DEFAULT_RANK)]
+public class VectorLayerToHostConverter : ISpeckleObjectToHostConversion, IRawConversion<VectorLayer, Task<string>>
+{
+  private readonly IConversionContextStack<Map, Unit> _contextStack;
+  private readonly IRawConversion<Base, ArcGIS.Core.Geometry.Geometry> _gisGeometryConverter;
+
+  public VectorLayerToHostConverter(
+    IConversionContextStack<Map, Unit> contextStack,
+    IRawConversion<Base, ArcGIS.Core.Geometry.Geometry> gisGeometryConverter
+  )
+  {
+    _contextStack = contextStack;
+    _gisGeometryConverter = gisGeometryConverter;
+  }
+
+  public object Convert(Base target) => RawConvert((VectorLayer)target);
+
+  public Task<string> RawConvert(VectorLayer target)
+  {
+    try
+    {
+      return QueuedTask.Run(() =>
+      {
+        var projectUtils = new ArcGISProjectUtils();
```

Comment:
Do we need new ones of these every time? is this our thing? If the latter can we inject it and interface it out plz?